src/_index.js

```diff
@@ -24,7 +24,8 @@ export class EasyTabAccordion{
             receiver: 'data-eta-receiver',
             hash: 'data-eta-hash',
             hashScroll: 'data-eta-hash-scroll',
-            animation: 'data-eta-animation'
+            animation: 'data-eta-animation',
+            preventDefault: 'data-eta-prevent-default',
```

Comment:
We will not create more attributes for the new options as we prefer settings via JSON instead.